Dockerfile.wheels

```diff
@@ -0,0 +1,15 @@
+FROM python:3.6-stretch
+
+COPY csv-generator /opt/csv-generator
```

Comment:
if these are really two separate applications, this can be optimized into two different Docker images so that changes to one of the wheels do not cause the other one to be re-built too. For later, unless this build takes a long time

Reply:
it seems to take 5 seconds for now so it's ok